app.json

```diff
@@ -25,6 +25,12 @@
     },
     "web": {
       "favicon": "./assets/favicon.png"
-    }
+    },
+    "extra": {
+      "eas": {
+        "projectId": "0ea8d500-a6ae-43f1-a262-51a05ccceed9"
+      }
+    },
+    "owner": "jeremywubv"
```

Comment:
is this because we have an expo portal login somewhere? if so, lmk and i'll make an RP one

Reply:
I dont know the r|p expo login so i made a demo project to test notifications